decidim-elections/spec/commands/decidim/elections/admin/unpublish_election_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+module Decidim::Elections::Admin
+  describe UnpublishElection do
+    subject { described_class.new(election, user) }
+
+    let!(:user) { create(:user, :admin, :confirmed, organization: participatory_process.organization) }
+    let!(:participatory_process) { create(:participatory_process, :with_steps) }
+    let(:step) { participatory_process.steps.first }
+    let!(:election) { create(:election, :published) }
```

Comment:
Here I think you can just use the `election` factory, without the `published` trait. 😊

Reply:
Also done!

Reply:
This stayed the same because we put `unpublished` as the default, instead of `published`